docs/reference/commandline/index.md

```diff
@@ -62,20 +62,20 @@ read the [`dockerd`](dockerd.md) reference page.
 | [events](events.md) | Get real time events from the server                   |
 | [exec](exec.md) | Run a command in a running container                       |
 | [export](export.md) | Export a container's filesystem as a tar archive       |
-| [kill](kill.md) | Kill a running container                                   |
+| [kill](kill.md) | Kill one or more running containers                        |
```

Comment:
Tempted to say that's best left to the actual reference page. I feel this page is to get a quick overview of available commands; e.g. a command to remove a container. Clicking through, I'll learn that I can also remove multiple containers in one go, or force removing, or use a filter, etc.